cluster/gce/container-linux/configure-helper.sh

```diff
@@ -1103,6 +1103,9 @@ function start-kube-controller-manager {
   if [[ -n "${SERVICE_CLUSTER_IP_RANGE:-}" ]]; then
     params+=" --service-cluster-ip-range=${SERVICE_CLUSTER_IP_RANGE}"
   fi
+  if [[ -n "${CONCURRENT_SERVICE_SYNCS:-}" ]]; then
```

Comment:
It might be useful to expose this variable inside cluster/gce/config-(default | test).sh.

Reply:
Sure, added to config-*.sh.